UI/PauseScreen.cpp

```diff
@@ -321,6 +321,9 @@ void GamePauseScreen::CreateViews() {
 	continueChoice->OnClick.Handle<UIScreen>(this, &UIScreen::OnBack);
 
 	std::string gameId = g_paramSFO.GetValueString("DISC_ID");
+	if (gameId.empty() || gameId == "UCJS10041") {
```

Comment:
Hmmm if they use made up gameID, then they shouldn't have problem with per-game config to begin with and that ID detection is now only a requirement for that one feature, it's unfortunately the most important one considering how bad can some homebrew work.
 I could change ID check to non iso/cso/pbp stored in memstick\PSP\GAME as cheats do now as it works fine in practice, I mean sure someone could throw full games in separate folders inside that place, but meh that would be worst case scenario and even then it would just cause their per-game config to use fake ID without affecting anything else.

 Fun fact ~ I checked that loco roco demo, and it's encrypted by keys we don't have. Guess it's exact reason why popular homebrew which are commonly signed as loco roco also have that problem.:]